internal/app/networkd/pkg/networkd/misc.go

```diff
@@ -12,6 +12,9 @@ import (
 	"net"
 	"strings"
 	"text/template"
+
+	"github.com/pkg/errors"
```

Comment:
We moved to standard library wrapping.

Reply:
Ah, yeah.  Wishing he would update his library to use stdlib with the same API.  Oh well.